primo/data_parser/well_data_columns.py

```diff
@@ -76,8 +76,9 @@ class WellDataColumnNames:  # pylint: disable=too-many-instance-attributes
     well_integrity: Optional[str] = None
 
     # Columns for efficiency metrics
-    elevation_delta: Optional[str] = None
-    dist_to_road: Optional[str] = None
+    avg_elevation_delta: Optional[str] = None
+    avg_dist_to_road: Optional[str] = None
+    population_density: Optional[str] = None
```

Comment:
Why does population density need to be an input column? We have utilities to include it from Census no?